options/locale/locale_en-US.ini

```
@@ -1583,8 +1583,14 @@ teams.write_permission_desc = This team grants <strong>Write</strong> access: me
 teams.admin_permission_desc = This team grants <strong>Admin</strong> access: members can read from, push to and add collaborators to team repositories.
 teams.repositories = Team Repositories
 teams.search_repo_placeholder = Search repository…
-teams.add_team_repository = Add Team Repository
+teams.add_team_repository = Add
 teams.remove_repo = Remove
+teams.add_all_repos = Add All
+teams.remove_all_repos = Remove All
+teams.remove_all_repos_title = Remove all team repositories
+teams.remove_all_repos_desc = This will remove all of the repositories from the team.
+teams.add_all_repos_title = Add all repositories
+teams.add_all_repos_desc = This will add all of the organizations repositories to the team.
```

```suggestion
teams.add_all_repos_desc = This will add all the organization's repositories to the team.
```
I'm not sure about this. _Someone who's a native speaker can suggest?_ 😄